google/cloud/storage/README.md

```diff
@@ -44,7 +44,29 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.2.x - TBD
+### v1.2.x - 2019-07
+
+** Breaking Changes **: we accidentally left two functions in the public API,
```

Comment:
Something's funny with the bolding and the use of asterisks here. Maybe related to the asterisk bullets and the ones for bolding?

Reply:
Thanks! Should be fixed now.